README.md

```diff
@@ -1,15 +1,15 @@
+1 | Polarization | <ul><li> Project group accommodates some rather than all representatives; favours appearances over outcomes.</ul></li><ul><li>Volume of work is prioritized over efficiency and organization.</ul></li><ul><li>CollaborationThe notion of collaboration exists in communications and operationally instead of as a wholesome project concept.</ul></li><ul><li>Feedback process follows antiquated methodology, is irregular, non-existant or biased - feedback is only being initiated if a large problem is brought forward or a large investment is involved</ul></li><ul><li>Feedback and areas of improvement from minorities are either minimalized, avoided, and/or ignored.</ul></li><ul><li>"Walking on Thin Ice" approach is common as potentially sensitive project elements, such as accessibility and demographic data, is regarded with fear of open conflict - inequalities and large data gaps are pravalent.</ul></li><ul><li>Product/Service data are collected from random sample(s) of users instead of all users who maintain, use and implement the product/service.</ul></li><ul><li>Records management is acknowledged and approached with simplicity rather than complexity such as consideration of electronic EDRMS folders, off-site records maintenance, different types of storage, etc.</ul></li>
+2 | Minimization | <ul><li>Product/Service's foundation is for everyone but depends on appearances for diversity, equity and inclusion.</ul></li><ul><li>Systems Thinking is basic within project group and the scope is inclusive to those within decision maker's purview.</ul></li><ul><li>Data is collected from management with the assumption that data collection is substantial & methodical and there is substantial dependence on socio-economic status for feedback.</ul></li><ul><li>The project group retention rate of miniorites is minimal because assimilation and generalization greatly influences working style and deliverables.</ul></li><ul><li> Complex feedback methodologies are considered and/or beginning with active and consentual collection and participation of users.</ul></li> <ul><li>Feedback of minorities are reworded for diplomacy and not seen as "quick wins".</ul></li><ul><li>Semantics, syntax and enuciation of product/service scripts, communications, and/or other related items depend on and celebrates mastery of the dominant group's language and vocabulary.</ul></li><ul><li>Combination of anti-oppression langauge and vocabulary is only considered when there is a complaint .</ul></li><ul><li>Product/Service collaboration between divisions and ministries are in forming stages, with few champions, and reputation building is underway.</ul></li>
```

Comment:
I'm personally offended by the word purview!  Can we find something a little easier to digest?

Reply:
& is only used here.  Go with and for consistency.

Reply:
Resolved.